pyvista/plotting/render_window_interactor.py

```diff
@@ -547,6 +547,110 @@ def enable_trackball_style(self):
         self._style_class = None
         self.update_style()
 
+    def enable_paraview_2d_style(self):
```

Comment:
I'm wondering if we should just call this `enable_2d_style()`?

Reply:
I'm +1 on this suggestion.  This also frees us from matching ParaView behavior 100%.  We can still mention that it is similar to the behavior in ParaView.

Reply:
Using this now in https://github.com/pyvista/pyvista/pull/5336/commits/4e38f0de087521ada6c7901e89caf27a72be353c